src/components/v5/shared/SearchSelect/hooks.ts

```diff
@@ -7,12 +8,12 @@ import { type SearchSelectOptionProps } from './types.ts';
 export const useSearchSelect = (
   items: SearchSelectOptionProps[],
   searchValue: string,
+  showSearchValueAsOption: boolean,
```

Comment:
do we need this property? is there any place where we wouldn't want to display the searched property? worst case, we can just filter the items from that component